lndr-backend/db/migrations/2018-10-08T10.00.00Z_change_push_data_pk.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE push_data DROP CONSTRAINT push_data_pkey;
+ALTER TABLE push_data ADD PRIMARY KEY (channel_id);
```

Comment:
Good, but still need to change the type of the `channel_id` column from `TEXT` to `CHAR(36)`. This might not be something we can safely do atomically.

The safest thing to do here would be:
 1. Add a new column with the desired data type.
 2. Copy data from the old column to the new column.
 3. Drop the old `channel_id` column.
 4. Rename the new column to `channel_id`.
 5. Change the primary key (as L1-L2 is doing already :+1:).

Reply:
Check out the latest update

Reply:
Please see the latest update, confirmed that this query runs successfully locally